discussions/api.py

```diff
@@ -0,0 +1,114 @@
+"""API for open discussions integration"""
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.db import transaction
+from open_discussions_api.client import OpenDiscussionsApi
+from open_discussions_api.constants import ROLE_STAFF
+
+from discussions.models import DiscussionUser
+from discussions.exceptions import DiscussionUserSyncException
+
+
+def get_staff_client():
+    """
+    Gets a client configured for user management
+    """
+    if not settings.OPEN_DISCUSSIONS_JWT_SECRET:
+        raise ImproperlyConfigured('OPEN_DISCUSSIONS_JWT_SECRET must be set')
+    if not settings.OPEN_DISCUSSIONS_BASE_URL:
+        raise ImproperlyConfigured('OPEN_DISCUSSIONS_BASE_URL must be set')
+    if not settings.OPEN_DISCUSSIONS_API_USERNAME:
+        raise ImproperlyConfigured('OPEN_DISCUSSIONS_API_USERNAME must be set')
+
+    return OpenDiscussionsApi(
+        settings.OPEN_DISCUSSIONS_JWT_SECRET,
+        settings.OPEN_DISCUSSIONS_BASE_URL,
+        settings.OPEN_DISCUSSIONS_API_USERNAME,
+        roles=[ROLE_STAFF]
+    )
+
+
+def create_or_update_discussion_user(user_id):
+    """
+    Create or update a DiscussionUser record and sync it
+
+    Args:
+        user_id (str): user id of the user to sync
+    """
+    discussion_user, _ = DiscussionUser.objects.get_or_create(user_id=user_id)
```

Comment:
Can you remove the `discussion_user, _ = ` part, since this instance of `discussion_user` is not used?

Reply:
It is on line 44.

Reply:
Oh right